ci_scripts/dashboard/common.cmake

```diff
@@ -0,0 +1,500 @@
+#------------------------------------------------------------------------------#
+# Distributed under the OSI-approved Apache License, Version 2.0.  See
+# accompanying file Copyright.txt for details.
+#------------------------------------------------------------------------------#
```

Comment:
this is not our license

Reply:
I see your license now, thanks.  Since I have modified all these scripts, and since the apache 2 license does not require a derivative work of the software, or modifications to the original, to be distributed using the same license, I'm just going to remove these two license statements.  I think this just results in re-distribution under the license in the root directory `LICENSE` file.